game/menu.cc

```diff
@@ -15,6 +15,10 @@ std::string MenuOption::getName() const {
 	return "";
 }
 
+std::string MenuOption::getVirtName() const {
+	return !virtualName.empty() ? virtualName : std::string();
```

Comment:
syntax is <statement> ? action on true : action on false.
so `!virtualName.empty() ? virtualName : std::string(); `completely reverses that which seems a little confusing, I'd rather have:

`return virtualName.empty() ? std::string() :  virtualName;`